speech/google/cloud/speech/_gax.py

```diff
@@ -104,9 +104,9 @@ def async_recognize(self, sample, language_code=None,
         audio = RecognitionAudio(content=sample.content,
                                  uri=sample.source_uri)
         api = self._gapic_api
-        response = api.async_recognize(config=config, audio=audio)
+        operation_future = api.async_recognize(config=config, audio=audio)
 
-        return Operation.from_pb(response, self)
+        return Operation.from_pb(operation_future.last_operation_data(), self)
```

Comment:
Why is this returning `Operation.from_pb(operation_future.last_operation_data())` and not just `operation_future`? The _entire point_ of the `_OperationFuture` class is that it has a `.result()` method and such to get the result (while `Operation` objects do not and have to be fed to a client.)